package.json

```diff
@@ -40,28 +40,25 @@
     },
     "dependencies": {
         "@azure/functions": "^1.2.3",
-        "@types/lodash": "^4.14.119",
-        "@types/node": "10.14.2",
-        "@types/uuid": "~3.4.4",

```

Comment:
No references found in *.d.ts files - moved to a dev dependency. Only result was a comment in the code, not an actual reference
<img width="1668" alt="Screen Shot 2022-04-22 at 2 44 45 PM" src="https://user-images.githubusercontent.com/11282622/164802571-f605295f-6774-4556-92b9-26e350f5b52e.png">